docs/site/src/pages/reference/python-client/write-data.mdx

```diff
@@ -12,14 +12,16 @@ directly to a channel, fetch a range and set its data, or leverage writers for
 writing large volumes of data.
 
 Writes in Synnax are more complicated than reads, and, as such, we recommend
-checking out our [concepts](../concepts/write) page to learn more about
-the best practices for writing data to Synnax.
+checking out our [concepts](/reference/concepts/write) page to learn more about
+the best practices for writing data to Synnax. 
+
+At the very least, you should be familiar with the [rules of writes](/reference/concepts/writes#rules-of-writes).
```

Comment:
The tone on this statement is off-putting and somewhat condescending. Why don't you just add the sentence "The rules of writes are especially important." to the end of the previous paragraph?